src/main/java/com/openshift/internal/restclient/ResourceFactory.java

```diff
@@ -104,7 +104,7 @@
 	}
 	private IClient client;
 	
-	public ResourceFactory(IClient client) {
+	protected ResourceFactory(IClient client) {
```

Comment:
Why are we changing the visibility?  It's labeled an internal package and if we are really concerned about extension maybe we should just mark it as final